### DISCUSSION TITLE
How do you get x-ClientVersion

### DISCUSSION THREAD

**Discussion by lewandowski-anthony** [daye10/dokkan-battle-farm-bot-app#29] [Q&A]
Hello,
Amazing work you've done here 🙏🏻 I'm trying to understand how does the game works by myself and was wondering how did you get the X-ClientVersion ? 
Thank you for your answer :)

**Comment by daye10** [comment]
Hello, you can DM on discord. To make it simple, you have to sniff your device requests.

**Reply by lewandowski-anthony** [reply]
Can you please give me your Discord ID please :) Mine is apasdelew

**Comment by daye10** [comment]
Added you 👍
